frontend/kesaseteli/employer/src/types/domain-object-type.d.ts

```diff
@@ -0,0 +1,6 @@
+import Application from 'kesaseteli/employer/types/application';
+import Company from 'kesaseteli/employer/types/company';
+
+type DomainObjectType = typeof Application | typeof Company | unknown;
```

Comment:
Doesn't this do the trick?
```suggestion
type DomainObjectType = Application | Company | unknown;
```

Reply:
Removed the whole file as it was not used anymore